web/app/js/components/NetworkGraph.jsx

```diff
@@ -105,7 +93,7 @@ export class NetworkGraphBase extends React.Component {
     let graphData = this.getGraphData();
 
     // check if graph is present to prevent drawing of multiple graphs
-    if (this.svg.select("circle")._groups[0][0]) {
+    if (this.svg.select("circle")._groups[0][0]) { // eslint-disable-line
```

Comment:
Ahhh, this one is extraneous, actually, since I eventually decided to disable the `no-underscore-dangle` rule globally, since we rely on _ prefixes and suffixes in a few places. Will remove the comment here.